lib/munster/controllers/receive_webhooks_controller.rb

```diff
@@ -2,55 +2,68 @@
 
 module Munster
   class ReceiveWebhooksController < ActionController::API
-    class HandlerRefused < StandardError
+    class InvalidRequest < StandardError
     end
 
     class HandlerInactive < StandardError
     end
 
-    def create
-      handler = lookup_handler(params[:service_id]).new
+    class UnknownHandler < StandardError
```

Comment:
They are convenient to place in the controller because they don't need a namespace to raise and to match, and this is the only spot where they get raised. Once we transition to a setup where these errors may get raised from the background job too - we'll move them into the gem namespace.